features/solvers.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Check for solvers available in Thoth
+    Scenario: Check there are minimum solvers available in Thoth
+        Given a set of solvers
```

Comment:
This feels strange, we are stating preconditions that we check for?! Shouldnt we simply start with the deployment being accessible and then we should get the list of solvers?

Reply:
```
Feature: Check for solvers available in Thoth
    Scenario: Check there are minimum solvers available in Thoth
        Given a set of solvers
            | solver_name |
            | solver-rhel-8-py36 |
            | solver-fedora-31-py37 |
            | solver-fedora-31-py38 |
        And deployment is accessible using HTTPS
        When we ask for the available solvers
        Then they should include at least the given solvers
```

The check for the deployment accessible comes right after, the first statement is a table which cannot fail. I can invert it :)

Reply:
I think we shoulds assume/have the first table, we simply query, and the results should match the table

Reply:
What about now?

```
Feature: Check for minimum solvers available in Thoth
    Scenario: Check there are minimum solvers available in Thoth
        Given deployment is accessible using HTTPS
        And a minimum set of solvers requested
            | solver_name |
            | solver-rhel-8-py36 |
            | solver-fedora-31-py37 |
            | solver-fedora-31-py38 |
        When we ask for the available solvers
        Then they should include at least the minimum set of solvers
```